pages/common/ifconfig.md

```diff
@@ -1,6 +1,7 @@
 # ifconfig
 
 > Network Interface Configurator.
+> More information: <https://manned.org/ifconfig>.
```

Comment:
Based on the Arch Linux package https://archlinux.org/packages/core/x86_64/net-tools/, this is the official source for the man page:
```suggestion
> More information: <https://net-tools.sourceforge.io/man/ifconfig.8.html>.
```
Should we use this instead?

Reply:
Sounds like a plan @navarroaxel